provider/scripts/spark/offline_store_spark_runner.py

```diff
@@ -303,9 +316,10 @@ def execute_sql_query(
         set_spark_configs(spark, spark_configs)
 
         if (
-            job_type == "Transformation"
-            or job_type == "Materialization"
-            or job_type == "Training Set"
+            job_type == JobType.TRANSFORMATION
```

Comment:
`job_type in valid_job_types`? 
When is this not true?

Reply:
Are you asking me to change this to `job_type in valid_job_types`?

Reply:
yeah, or list(JobType).

My point is if someone adds a new jobtype do they have to add to this condition too? If so, you've increased cognitive load and this is prone to errors when changes are made

Reply:
but what if we add a job type that we don't want in this?

Reply:
Can make a sublist of valid things for this function in that case